ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/UPLINK_P2P_VRFS_TESTS_SPINE1.cfg

```diff
@@ -0,0 +1,85 @@
+!RANCID-CONTENT-TYPE: arista
+!
+vlan internal order ascending range 1006 1199
+!
+transceiver qsfp default-mode 4x10G
+!
+service routing protocols model multi-agent
+!
+hostname UPLINK_P2P_VRFS_TESTS_SPINE1
+!
+spanning-tree mode none
+!
+no enable password
+no aaa root
+!
+vrf instance MGMT
+!
+interface Ethernet1
+   description P2P_LINK_TO_UPLINK_P2P_VRFS_TESTS_LEAF1_Ethernet1
+   no shutdown
+   mtu 9214
+   mac security profile TEST
+   no switchport
+   ip address 10.42.42.8/31
+!
+interface Ethernet1.42
+   description P2P_LINK_TO_UPLINK_P2P_VRFS_TESTS_LEAF1_Ethernet1.42 vrf PROD
+   no shutdown
+   mtu 9214
+   encapsulation dot1q vlan 42
+   vrf PROD
+   ip address 10.42.42.8/31
+!
+interface Ethernet1.66
+   description P2P_LINK_TO_UPLINK_P2P_VRFS_TESTS_LEAF1_Ethernet1.66 vrf ONLY-ON-L2LEAF
+   no shutdown
+   mtu 9214
+   encapsulation dot1q vlan 66
+   vrf ONLY-ON-L2LEAF
+   ip address 10.42.42.8/31
+!
+interface Loopback0
+   description Router_ID
+   no shutdown
+   ip address 192.168.42.1/32
+!
+ip routing
+no ip routing vrf MGMT
+!
+router bgp 65000
+   router-id 192.168.42.1
+   maximum-paths 4 ecmp 4
+   update wait-install
+   no bgp default ipv4-unicast
+   neighbor IPv4-UNDERLAY-PEERS peer group
+   neighbor IPv4-UNDERLAY-PEERS send-community
+   neighbor IPv4-UNDERLAY-PEERS maximum-routes 12000
+   neighbor 10.42.42.9 peer group IPv4-UNDERLAY-PEERS
+   neighbor 10.42.42.9 remote-as 65000
+   neighbor 10.42.42.9 description UPLINK_P2P_VRFS_TESTS_LEAF1_Ethernet1
+   redistribute connected
+   !
+   address-family ipv4
+      neighbor IPv4-UNDERLAY-PEERS activate
+   !
+   vrf ONLY-ON-L2LEAF
+      router-id 192.168.42.1
+      neighbor 10.42.42.9 remote-as 65000
+      neighbor 10.42.42.9 peer group IPv4-UNDERLAY-PEERS
+      neighbor 10.42.42.9 description UPLINK_P2P_VRFS_TESTS_LEAF1_Ethernet1.66
```

Comment:
Should we also add vrf to this description?

Reply:
it is just above so don't think so

Reply:
The description is also visible in various CLI outputs and syslogs, so I think it could be useful.

Reply:
added like under subinterface